src/shared/codelens/typescriptCodeLensProvider.ts

```diff
@@ -114,11 +113,11 @@ export function initialize({
             })
             const runtime = CloudFormation.getRuntime(resource)
 
-            if (params.isDebug && unsupportedNodeJsRuntimes.has(runtime)) {
+            if (unsupportedNodeJsRuntimes.has(runtime)) {
```

Comment:
Would a whitelist be more robust than a blacklist here? What if the user's SAM Template specifies a runtime like `nodejs6.11`?